jenkins/test_runner_image.yml

```diff
@@ -5,7 +5,7 @@
     msg: |
       Running the following image test cases:
       {% for test_case in osbuild_composer_image_test_cases %}
-        - {{ osbuild_composer_image_test_cases }}
+        - {{ test_case | splitext | first }}
```

Comment:
For future reference, would be nice to put the description "Fix the debug loop variable that shows the list of image tests two times rather than showing them individually man_facepalming" in the commit message rather than (only) in the PR description, as the commit messages stay around in the git history.

Reply:
@teg Good point. I forgot to do that as I stacked them up. :/